### ISSUE TITLE
Is this project still updated?

### ISSUE THREAD

**Issue by kcrazy** [redsift/ingraind#309]
Hi,
Is this project still updated? 
or anything else planned?

**Comment by rsdy** [comment]
Not very much. Most of the focus is on [redbpf](https://github.com/foniod/redbpf) development, since it seems to scale better for most people's needs.

**Comment by RobertKielty** [comment]
@rsdy do you mean https://github.com/foniod/redbpf  ?

**Comment by rsdy** [comment]
@RobertKielty yes, although since that message redbpf development has slowed down a bit too.
